app/models/offspring.rb

```diff
@@ -0,0 +1,7 @@
+# Offsprings are related to a user and only one user.
+# They can have one shift assingned
+class Offspring < ApplicationRecord
+  validates :first_name, presence: true, length: { in: 3..60 }
+  validates :last_name, presence: true, length: { in: 3..60 }
+  validates :age, presence: true, numericality: { only_integer: true, greater_than: 5, less_than: 11 }
```

Comment:
This makes no sense at all, age should be definable in some other place without touching the model, defining 5..11 will make it very restrictive and defining 9.99 will not achieve anything.